music21/test/musicXMLTest.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+import unittest
+from xml.etree import ElementTree as ET
+
+import music21 as m21
+
+DATA_FILE = Path(__file__).parent / 'data/Test1.mxl'
+
+class Test(unittest.TestCase):
+    def testDeserialization(self):
```

Comment:
these method names could be more descriptive since (de)serialization is the point of the entire module! :-)